indico/modules/events/management/templates/clone_event.html

```diff
@@ -91,8 +91,13 @@
                             {% set count %}
                                 <span class="count"></span>
                             {% endset %}
-                            {% trans link='<a class="event-list">'|safe, endlink="</a>"|safe -%}
-                                {{ count }} new events will be created. {{ link }}When?{{ endlink}}
+                            {% trans link='<a class="event-list">'|safe, endlink='</a>'|safe,
+                                     lastday='<span class="last-day">'|safe, endlastday='</span>'|safe -%}
+                                {{ count }} new events will be created.
+                                {{ lastday }}
```

Comment:
I think you can strip whitespace on the right side here (`-}}`), and on the left side of `endlastday`.